tags.go

```diff
@@ -25,3 +27,10 @@ type TagService interface {
 	// Lookup returns the set of tags referencing the given digest.
 	Lookup(ctx context.Context, digest Descriptor) ([]string, error)
 }
+
+// TagIndexes proves method to retreive all the digests that a tag historically pointed to
```

Comment:
naming nit: why is this called an index? I know that term gets used at the storage level but not sure if it fits here. These could only be created through the manifests endpoints and it kind of a manifest history. Maybe referring to these are manifests rather than indexes would be clearer.

Reply:
Yeah. I think indexes is probably not the right term here. I was trying to stick to what is in storage. I think just `ManifestDigests` might be better. wdyt?